ui-src/parser/creators/createColorsLibrary.ts

```diff
@@ -0,0 +1,41 @@
+import { sleep } from '@plugin/utils/sleep';
+
+import { sendMessage } from '@ui/context';
+import { PenpotFile } from '@ui/lib/types/penpotFile';
+import { uiColorLibraries } from '@ui/parser/libraries/UiColorLibraries';
+
+export const createColorsLibrary = async (file: PenpotFile) => {
+  let librariesBuilt = 1;
+  const libraries = uiColorLibraries.all();
+
+  sendMessage({
+    type: 'PROGRESS_TOTAL_ITEMS',
+    data: libraries.length
+  });
+
+  sendMessage({
+    type: 'PROGRESS_STEP',
+    data: 'libraries'
+  });
+
+  for (const library of libraries) {
+    for (let i = 0; i < library.fills.length; i++) {
```

Comment:
avoid using 1 length variables, maybe is better to call it index?

Reply:
Fixed